mount-daemon/controllers/clientmount_controller.go

```diff
@@ -188,6 +188,11 @@ func (r *ClientMountReconciler) unmount(ctx context.Context, clientMountInfo dws
 		}
 	}
 
+	// Remove the mount directory. It's not a big deal if this fails, so we just log a failure and don't return it
+	if err := r.rmdir(clientMountInfo.MountPath); err != nil {
+		log.Error(err, "Unable to remove mount directory", "Path", clientMountInfo.MountPath)
```

Comment:
Shouldn't you return after this so you don't get the log.Info() below?

Reply:
This error is only about deleting the directory where the mount existed. The mount has already been unmounted, and we're not doing to retry deleting the directory. I think it makes sense to leave the log.Info below even if the rmdir fails.

Reply:
Yea, nevermind.